bindata/network/ovn-kubernetes/managed/alert-rules.yaml

```diff
@@ -329,3 +328,12 @@ spec:
       for: 10m
       labels:
         severity: critical
+    # OVN northd performance alerts
+    - alert: OVNKubernetesNorthdCPUUsageHigh
+      annotations:
+        summary: OVN northd {{ $labels.instance }} is greater than {{ $value }} percent CPU usage for a period of time.
+        description: High OVN northd CPU usage indicates high load on the networking control plane.
+      expr: (sum(rate(container_cpu_usage_seconds_total{container="northd"}[5m])) BY (instance, name) * 100) > 80
```

Comment:
so over 15 minutes the northd CPU should not average over 80% cpu? @dceara seem reasonable?

Reply:
Over 20 minutes actually because of the expression.

Reply:
I don't think that description is correct here. The problem is that northd almost always re-computes the whole network topology every time even small change is made.  On a high-scale cluster it may take several seconds per one run.  Assuming some changes are made in the high-scale cluster every few seconds, northd may stay at 100% CPU usage for a really long time.  And it will take almost the same time for it to process a very small change and a really big one.  So, the alert doesn't really reflect any issues with northd or other components, especially at scale.  And it may produce a lot of false-positives.

Reply:
It really depends on the workload/cluster configuration.  I'm curious if @mohit-sheth has some ovn-northd CPU usage data from recent node-density/density-cni runs with larger scale clusters (>= 120 nodes).

Reply:
Accept the description is incorrect.
I tested it with nodes > 120 after scale testing runs.
I think you raise a reasonable point though that this needs better testing.
I will remove the CPU usage from this PR and introduce it later following more tests.